flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/metadata/MetadataTestUtil.scala

```diff
@@ -298,6 +301,124 @@ object MetadataTestUtil {
       )
   }
 
+  private def createTableSourceTable1(): Table = {
+    val catalogTable = CatalogTable.fromProperties(
+      Map(
```

Comment:
You don't need to go through properties here. You can simply call `CatalogTable.of()` with concrete instances.

Reply:
Added a hotfix commit to address that, (clean up also existing code)

Reply:
Could you also update the `MetadataTestUtil#createProjectedTableSourceTable`

Reply:
Done, thx, I missed it.